.travis.yml

```diff
@@ -19,10 +23,12 @@ install:
 # Assist with ci test debugging:
 #   - DEBUG=1
 before_script:
-    - image="kylemanna/openvpn"
-    - docker build -t "$image" .
+# prepare qemu
+    - if [ $ARCH == 'aarch64' ]; then docker run --rm --privileged multiarch/qemu-user-static:register --reset; fi
+    - image="$IMAGE"
+    - docker build -f $DOCKERFILE -t "$image" .
     - docker inspect "$image"
-    - docker run --rm "$image" openvpn --version || true # why does it return 1?
+    - docker run --rm "$image" openvpn --version || true; # why does it return 1?
```

Comment:
This line is extraneous

Reply:
it was in the original travis file. I'm happy to remove it.